src/resources/Correction/BenefitCompanyStatementResource.ts

```diff
@@ -1,11 +1,13 @@
 import { CorrectionTypes, FilingCodes, NameRequestTypes } from '@/enums/'
 import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module/'
+import { ResourceIF } from '@/interfaces/'
 
-export const BenefitCompanyStatementResource: any = {
-  addressLabel: 'Registered Office',
-  entityType: CorpTypeCd.BENEFIT_COMPANY,
+export const BenefitCompanyStatementResource: ResourceIF = {
   entityReference: 'Company',
   contactLabel: 'Registered Office',
+  displayName: null, // not used
+  entityType: CorpTypeCd.BENEFIT_COMPANY,
+  addressLabel: 'Registered Office',
```

Comment:
I changed this to be in the same order as the interface (and the other resource files).